scania-digital-design-system/sdds
# fix(dropdown): add openDirection prop to filter version

**Describe pull-request**  
Added missing openDirection prop to filter version of the dropdown component.

**Solving issue**  
Fixes: [DTS-1473](https://tegel.atlassian.net/jira/software/projects/DTS/boards/1?selectedIssue=DTS-1473)

**How to test**  
1. Go to Storybook link below
2. Check in Dropdown Filter -> Default
3. Try out the "Open direction" control

**Suggested test steps**

[DTS-1473]: https://tegel.atlassian.net/browse/DTS-1473?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
